docs/software/dns_servers.md

```diff
@@ -116,6 +116,31 @@ Pi-hole is a DNS sinkhole with web interface that will block ads for any device
     - [The Big Blocklist Collection by `WaLLy3K`](https://firebog.net/)
     - [Phishing Army blocklist](https://phishing.army/)
     - [Whitelist collection by `anudeepND`](https://github.com/anudeepND/whitelist)
+    - [RPiList](https://github.com/RPiList/specials/blob/master/Blocklisten.md): Block- and whitelists with focus on german domains
```

Comment:
Also when used as adjectives, in English, countries are always starting with capital letter 🙂:
```suggestion
    - [RPiList](https://github.com/RPiList/specials/blob/master/Blocklisten.md): Block- and whitelists with focus on German domains
```